spec/acceptance/class_spec.rb

```diff
@@ -2,16 +2,26 @@
 
 case fact('osfamily')
 when 'Debian'
-  package_name     = 'fail2ban'
-  config_file_path = '/etc/fail2ban/jail.conf'
-  service_name     = 'fail2ban'
+  package_name          = 'fail2ban'
+  config_file_path      = '/etc/fail2ban/jail.conf'
+  service_name          = 'fail2ban'
+  ssh_log_file          = '/var/log/auth.log'
+  ssh_jail              = 'ssh'
 when 'RedHat'
-  package_name     = 'fail2ban'
-  config_file_path = '/etc/fail2ban/jail.conf'
-  service_name     = 'fail2ban'
+  package_name          = 'fail2ban'
+  config_file_path      = '/etc/fail2ban/jail.conf'
+  service_name          = 'fail2ban'
+  ssh_log_file          = '/var/log/secure'
+  ssh_jail              = 'ssh'
+  # EPEL needs to be installed, otherwise it won't work
+  shell('wget http://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm')
```

Comment:
Missed this, should not have been merged.

Reply:
By default ensuring the package `epel-release` is installed works on CentOS since it's carried in the extras repo.